### PR TITLE
Specify that Exporter is experimental

### PR DESCRIPTION
**Pull request by tanmay-db**
## Changes

Exporter is experimental and although we mention that in https://registry.terraform.io/providers/databricks/databricks/latest/docs/guides/experimental-exporter but we don't have mention of that in our repository. Adding message. 

## Tests

Ran: `./terraform-provider-databricks exporter` and Warning is printed.
image

- [ ] `make test` run locally
- [ ] relevant change in `docs/` folder
- [ ] covered with integration tests in `internal/acceptance`
- [ ] relevant acceptance tests are passing
- [ ] using Go SDK

### CODE REVIEW

**Review by mgyucht — APPROVED** [review]
LGTM. Let's wait for @alexott to review as well before merging.

File: exporter/command.go
Comment (alexott) [inline exporter/command.go:79]: I'm not sure that adding this to the code is required... Exporter functionality isn't something that people just find in the docs, and it's already covered in the docs. Otherwise we'll need to add such warnings to most of our tooling - CLI, SDKs, etc.

Comment (tanmay-db) [reply exporter/command.go:79]: We want to highlight that it is experimental so it would also be nice to have this in the code.

**Pull request merged**
